src/boot/efi/secure-boot.c

```diff
@@ -11,3 +14,121 @@ BOOLEAN secure_boot_enabled(void) {
 
         return !EFI_ERROR(err) && secure;
 }
+
+static EFI_STATUS setup_mode_active(BOOLEAN *active) {
```

Comment:
doesn't matter, but we usually name return parameters `ret_xyz`, i.e. maybe rename `active` → `ret_active` here